packages/analytics/dashboard-renderer/src/components/BarChartRenderer.vue

```diff
@@ -4,15 +4,17 @@
     :query="query"
     :query-ready="queryReady"
   >
-    <AnalyticsChart
-      :chart-data="data"
-      :chart-options="options"
-      chart-title=""
-      :height="`${height}px`"
-      legend-position="bottom"
-      :show-annotations="chartOptions.showAnnotations"
-      tooltip-title=""
-    />
+    <div :style="{ height: `${height}px`}">
```

Comment:
Instead of adding inline styles, you can use `v-bind` down in the style block https://vuejs.org/api/sfc-css-features.html#v-bind-in-css